docker/nginx/default.conf

```diff
@@ -22,18 +22,36 @@ server {
     send_timeout                300;
 
     location /static/ {
+        if ($http_x_forwarded_proto = "http") {
+            return 301 https://$host$request_uri;
+        }
+        if ($http_x_forwarded_proto = "https") {
+            add_header Strict-Transport-Security "max-age=3600;" always;
```

Comment:
I suppose https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/ applies here . I could use a map over `$http_x_forwarded_proto` - but it's not clear how to only apply it to https requests. 

Alternatively, we could ignore the rfc and just add it on all requests...